### PR TITLE
(maint) Set leatherman version for downstream find_package calls

### PR DESCRIPTION
**Pull request by ajroetker**
This commit sets the leatherman version for the project and adds a version
file for cmake so that downstream projects can declare their version
dependency for leatherman in their find_package call for leatherman.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -98,7 +98,11 @@ if (LEATHERMAN_INSTALL)
     install(FILES ${CMAKE_FILES} DESTINATION "${INSTALL_LOC}/cmake/")
 
     configure_file(LeathermanConfig.cmake.in "${PROJECT_BINARY_DIR}/LeathermanConfig.cmake" @ONLY)
-    install(FILES "${PROJECT_BINARY_DIR}/LeathermanConfig.cmake" DESTINATION ${INSTALL_LOC})
+    configure_file(LeathermanConfigVersion.cmake.in "${PROJECT_BINARY_DIR}/LeathermanConfigVersion.cmake" @ONLY)
+    install(FILES
+    "${PROJECT_BINARY_DIR}/LeathermanConfig.cmake"
```
Comment (MikaelSmith) [inline CMakeLists.txt]: I'd like to see these indented 4 spaces.

**Comment by MikaelSmith** [comment]
Cool!

**Comment by MikaelSmith** [comment]
Kicking Travis.

**Comment by ajroetker** [comment]
@MikaelSmith added indentation in the CMake file

**Pull request merged**